src/components/ImageGalleryModal/ImageGalleryModal.jsx

```diff
@@ -0,0 +1,200 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
```

Comment:
I thought we had a lint rule about using `classnames` for this package on import.

Reply:
Hmm, there was no linting error for me at least. But I will change into `classnames`.

Reply:
Yeah, the eslint rule isn't in yet but we would like to standardize on the fully lowercase name. https://github.com/carbon-design-system/carbon-addons-iot-react/issues/1325